proto/tendermint/abci/types.proto

```diff
@@ -395,8 +403,8 @@ message ExecTxResult {
   bytes          data       = 2;
   string         log        = 3; // nondeterministic
   string         info       = 4; // nondeterministic
-  int64          gas_wanted = 5;
-  int64          gas_used   = 6;
+  int64          gas_wanted = 5 [json_name = "gas_wanted"];
```

Comment:
This changed and break-check is pointing it out. It was done automatically after updating the Makefile

Reply:
awesome, danke